.gitignore

```diff
@@ -35,3 +35,4 @@ ruby/lib/lib
 
 Gemfile.lock
 .dirstamp
+/.idea
```

Comment:
I usually prefer this kind of ignore to be put into your own personal, global `~/.gitignore` file instead of something that lives in each project.

Reply:
Ah, sure.